api/handler/get-survey.js

```diff
@@ -1,3 +1,4 @@
+/* eslint no-loop-func: 0, max-nested-callbacks: [2, 3]  */
```

Comment:
Oops. Looks like i just copied it from other file and did not edit it.
I will handle this is in the next merge task I am about to push.



Thanks,
Pooja

On Wed, Nov 18, 2015 at 3:56 PM, Christian Murphy <notifications@github.com>
wrote:

> In api/handler/get-survey.js
> <https://github.com/ser515asu/PRP-Manhattan-Project/pull/101#discussion_r45274060>
> :
>
> > @@ -1,3 +1,4 @@
> > +/* eslint no-loop-func: 0, max-nested-callbacks: [2, 3]  */
>
> @poojaRal <https://github.com/poojaRal> Why is no-loop-func disabled for
> this file?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ser515asu/PRP-Manhattan-Project/pull/101/files#r45274060>
> .
>